### PR TITLE
Cleanup more u32 being used as pointers.

### PR DESCRIPTION
**Pull request by louist103**

### CODE REVIEW

**Review by fig02 — APPROVED** [review]

File: include/z64.h [hunk 1]
```diff
@@ -871,12 +871,12 @@ typedef struct {
...
     /* 0x01 */ u8    format; // 1 = single, 2 = multi
-    /* 0x04 */ void* dlist;
+    /* 0x04 */ Gfx* dlist;
     union {
         struct {
-            /* 0x08 */ u32   source;
+            /* 0x08 */ void*   source;
             /* 0x0C */ u32   unk_0C;
-            /* 0x10 */ u32   tlut;
+            /* 0x10 */ void*   tlut;
```
Comment (Dragorn421) [inline include/z64.h:879]: Can you align the field names 😊

**Review by Dragorn421 — CHANGES_REQUESTED** [review]

File: include/z64.h [hunk 2]
```diff
@@ -871,12 +871,12 @@ typedef struct {
 typedef struct {
     /* 0x00 */ u8    type;
     /* 0x01 */ u8    format; // 1 = single, 2 = multi
-    /* 0x04 */ void* dlist;
+    /* 0x04 */ Gfx* dlist;
```
Comment (Dragorn421) [inline include/z64.h:874]: 😊

**Comment by fig02** [comment]
something got messed up while you were updating variable names

File: include/z64scene.h [hunk 3]
```diff
@@ -270,37 +270,37 @@ typedef struct {
 } LightSettings; // size = 0x16
 
 typedef struct {
-    /* 0x00 */ u8 count; // number of points in the path
+    /* 0x00 */ u8 count;      // number of points in the path
     /* 0x04 */ Vec3s* points; // Segment Address to the array of points
-} Path; // size = 0x8
+} Path;                       // size = 0x8
```
[the remark this answers is not in the document]
Comment (Dragorn421) [reply include/z64scene.h:275]:
```suggestion
} Path; // size = 0x8
```

**Review by Roman971 — APPROVED** [review]

**Pull request merged**